### PR TITLE
Global: Responsive Display & Flexbox Props

### PR DESCRIPTION
**Pull request by RobGentile17**
Screen Shot 2022-05-11 at 3 54 17 PM
Screen Shot 2022-05-11 at 3 54 37 PM
Screen Shot 2022-05-11 at 3 54 28 PM

This PR:
Responsive flexbox props works in rails but needs to be tweaked in react. 
All flexbox props were added to typescript  for convenience so those need to be tested:
- [x] Flex Direction
- [x] Flex Wrap 
- [x] Justify Content
- [x] Justify Self
- [x] Align Items 
- [x] Align Content
- [x] Align Self
- [x] Flex 
- [x] Flex Grow
- [x] Flex Shrink
- [x] Order

**Note** Due to the large number of changes in https://github.com/powerhome/playbook/pull/1846 conflicting with this PR originally, BOTH PRs exist here in this PR. The aforementioned PR will be closed after this one is merged.

#### Breaking Changes

No - new feature

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/PLAY-4

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [ ] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

**Review by thestephenmarshall — COMMENTED** [review]
I moved the work you did over from `globalProps.js` to `globalProps.ts` in [this PR](https://github.com/powerhome/playbook/pull/1796#issuecomment-1045328903) fwiw. I tested things out with `_image.tsx` from that side.

**Comment by thestephenmarshall** [comment]
Types have been updated from `master` and ordered alphabetically. The PR needs another look and round of testing to ensure it is ready.

**Review by thestephenmarshall — APPROVED** [review]

**Review by jasoncypret — APPROVED** [review]
WOW great job @thestephenmarshall 🚀

File: playbook/app/pb_kits/playbook/tokens/_display.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+$display_inline: inline;
+$display_block: block;
+$display_inline_block: inline-block;
+$display_flex: flex;
+$display_none: none;
+$display_inline_flex: inline-flex;
+$displays: (
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/tokens/_display.scss:7]: Nice 👍

File: playbook/app/pb_kits/playbook/utilities/_align_content.scss [hunk 2]
```diff
@@ -1,29 +1,10 @@
...
-  display: flex;
-  align-content: space-around !important;
-}
-
-.align_content_space_evenly {
-  display: flex;
-  align-content: space-evenly !important;
-}
\ No newline at end of file
+$align_content_values: (
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/utilities/_align_content.scss:1]: @RobGentile17 you're gonna love all this cleanup and simplification!

Comment (RobGentile17) [reply playbook/app/pb_kits/playbook/utilities/_align_content.scss:1]: I can't wait for it to be live and in use!

**Comment by jasperfurniss** [comment]
🚀

**Pull request merged**